src/app/modules/todo/pages/list-todo-page/list-todo-page.component.ts

```diff
@@ -43,28 +46,35 @@ export class ListTodoPageComponent implements OnInit, OnDestroy {
       .pipe(
         takeUntil(this.destroy$)
       )
-      .subscribe();
+      .subscribe((res) => res, (err) => {
```

Comment:
IMPORTANT!!! always unsubscribe a stream, why and how ? do a search :D. And you should create an abstract class to do that for all subscription